pmd-java/src/main/java/net/sourceforge/pmd/lang/java/JavaLanguageHandler.java

```diff
@@ -20,6 +20,12 @@ public JavaLanguageHandler(int jdkVersion, boolean preview) {
         this.preview = preview;
     }
 
+
+    public final int getJdkVersion() {
```

Comment:
this seems new… and unused…

Reply:
Yeah I think this comes from somewhere else.